README.md

```diff
@@ -50,9 +50,9 @@ please see the [neural network guide](docs/NeuralNetworkGuide.md).
 - Core ML specification is fully described in a set of protobuf files.
 They are all located in the folder `mlmodel/format/`
 - For an overview of the Core ML framework API, see [here](https://developer.apple.com/documentation/coreml).
-- To find the list of model types supported by Core ML, see [this](https://github.com/aseemw/coremltools/blob/f95f9b230f6a1bd8b0d9ee298b78d7786e3e7cfd/mlmodel/format/Model.proto#L229)
+- To find the list of model types supported by Core ML, see [this](https://github.com/apple/coremltools/blob/1fcac9eb087e20bcc91b41bc938112fa91b4e5a8/mlmodel/format/Model.proto#L229)
```

Comment:
Is there a way we can get a permanent URL for this? This will break each time we do an update right?

Reply:
This I believe is the permanent url, when we copy a link from Github, there are two options, I choose the permanent one.